src/clusterfuzz/_internal/tests/appengine/handlers/base_handler_test.py

```diff
@@ -146,7 +146,7 @@ def test_forbidden_not_logged_in(self):
     app = webtest.TestApp(flaskapp)
     response = app.get('/', expect_errors=True)
     self.assertEqual(response.status_int, 302)
-    self.assertEqual('http://localhost/login?dest=http%3A%2F%2Flocalhost%2F',
+    self.assertEqual('/login?dest=http%3A%2F%2Flocalhost%2F',
```

Comment:
Do we know why this has changed?

Reply:
Default behavior on flask, that changed among versions. Both are valid, see https://datatracker.ietf.org/doc/html/rfc7231#section-7.1.2